manifest: add support for `buildah manifest exists` (#4220)
containers/buildah: docs/buildah-manifest-exists.1.md

% buildah-manifest-exists(1)

## NAME
buildah\-manifest\-exists - Check if the given manifest list exists in local storage

## SYNOPSIS
**buildah manifest exists** *manifest*

## DESCRIPTION

> **buildah manifest exists** checks if a manifest list exists on local storage. Podman will

Reviewer: ```suggestion
**buildah manifest exists** checks if a manifest list exists in local storage. Buildah will
```
